Backend/lib/payment/src/Lib/Payment/Storage/Tabular/PaymentOrder.hs

```diff
@@ -58,6 +58,8 @@ mkPersist
       mandateMaxAmount HighPrecMoney Maybe
       mandateStartDate UTCTime Maybe
       mandateEndDate UTCTime Maybe
+      bankErrorCode Text Maybe
+      bankErrorMessage Text Maybe
```

Comment:
You didn't create migration for these changes. By the way these tabular types are the same for bpp and bap, so both DBs should be migrated

Reply:
Sure @roman-bodavskiy Will resolve the issues with migrations